scipy/optimize/tests/test_linprog.py

```diff
@@ -1001,6 +1001,17 @@ def test_bug_8664(self):
                           method=self.method)
         assert_(not res.success, "incorrectly reported success")
 
+    def test_bug_8973(self):
```

Comment:
Maybe it is worth including a more descriptive name, and point to issue #8973 in comments. Something like:

```
def test_constant_func_unbounded_region(self):
    """Test linear programming for constant objective functions and an unbounded feasible region.

    Test if bug described in https://github.com/scipy/scipy/issues/8973 was fixed."""
```

Reply:
@antonior92 Did you mean to include the first part about constant objective function?